client/src/components/questionsAnswers/D-QuestionsCards.jsx

```diff
@@ -108,19 +117,21 @@ function QuestionsCards(props) {
           />
         ))}
       </div>
-      <button
-        type="button"
-        onClick={() => {
-          if (moreAs === false) {
-            setNoAs(answers.length);
-          } else {
-            setNoAs(2);
-          }
-          setMoreAs(!moreAs);
-        }}
-      >
-        {moreAs ? 'Collapse answers' : 'Load more answers'}
-      </button>
+      {answers.length >= 2 ? (
+        <button
+          type="button"
+          onClick={() => {
+            if (moreAs === false) {
+              setNoAs(answers.length);
+            } else {
+              setNoAs(2);
+            }
+            setMoreAs(!moreAs);
+          }}
+        >
+          {moreAs ? 'Collapse answers' : 'Load more answers'}
+        </button>
+      ) : (<br />)}
```

Comment:
I could be wrong, but it looks like the paren and bracket are not lining up

Reply:
The opening bracket corresponding to the closing bracket you see there on line 134 is all the way up on line 119.  It lines up :)

Reply:
got cha